warp10/src/main/java/io/warp10/continuum/Tokens.java

```diff
@@ -253,7 +253,27 @@ public static ReadToken extractReadToken(String token) throws WarpScriptExceptio
     
     List<String> owners = new ArrayList<String>();
     List<String> producers = new ArrayList<String>();
-    
+    Map<String, String> labels = new HashMap<String, String>();
+
+    if (rtoken.getLabelsSize() > 0) {
+      labels = rtoken.getLabels();
+      if (!labels.isEmpty())
```

Comment:
Can you change the style for this file, opening braces are to appear on the same line as the for/if